dev/app/app.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 /**
+ * Change to switch between local JSON and firebase
+ */
+var localData = true;
+
```

Comment:
When you're working on `dev` then yes it can be changed. I'm not sure how to enforce a rule to always set it back to `true` before committing. Maybe it is something we can do with a test or using travis-ci.

If we want the option of using local files with `dist` then I guess one solution would be to create a separate `build` task for deployment and one for viewing the `dist` version.